packages/brand-positioning-block/src/components/Toolbar.tsx

```diff
@@ -0,0 +1,36 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { joinClassNames } from '@frontify/guideline-blocks-shared';
+import { IconArrowMove16, IconTrashBin16 } from '@frontify/fondue';
+import { ToolbarProps } from '../types';
+
+export const Toolbar = ({ draggableAttributes, draggableListeners, isDragging }: ToolbarProps) => {
+    const handleRemoveSelf = () => {
+        /** Todo */
+    };
+
+    return (
+        <div className="tw-flex tw-justify-end">
+            <div className="tw-bg-white tw-text-box-selected-inverse tw-flex tw-flex-shrink-0 tw-gap-[2px] tw-px-[1px] tw-spacing tw-items-center tw-h-[28px] tw-self-start tw-border tw-border-box-selected-inverse tw-rounded-[4px]">
+                <button
+                    {...draggableAttributes}
+                    {...draggableListeners}
+                    className={joinClassNames([
+                        'tw-bg-base  tw-inline-flex  tw-items-center tw-justify-center tw-w-[24px] tw-h-[24px] tw-rounded-sm',
```

Comment:
```suggestion
                        'tw-bg-base tw-inline-flex tw-items-center tw-justify-center tw-w-[24px] tw-h-[24px] tw-rounded-sm',
```

Reply:
Changed to all of your suggestions except: Please use multiple of 4 only for px value for the positioning of toolbar.

I got the feedback from Martin that the toolbar's border should be exactly in line with the item outline, and that is 3px. I will try to come up with a solution where it is a multiple of four, or maybe you have a rough idea?